### PR TITLE
docs: add model support

### PR DESCRIPTION
**Pull request by shan-mx**

### CODE REVIEW

**Comment by ZiniuYu** [comment]
onnx models are all uploaded to s3

**Comment by numb3r3** [comment]
According to this PR, I feel the model table contains some redundant information. Let's first hold this PR until we have an idea how to proceed.

File: docs/user-guides/server.md [hunk 1]
```diff
@@ -61,23 +61,38 @@ The procedure and UI of ONNX and TensorRT runtime would look the same as Pytorch
```
Comment (ZiniuYu) [inline docs/user-guides/server.md:61]:
```suggestion
(model-support)=
```

File: docs/user-guides/server.md [hunk 2]
```diff
@@ -249,7 +264,7 @@ For all backends, you can set the following parameters via `with`:
 
 | Parameter               | Description                                                                                                                    |
 |-------------------------|--------------------------------------------------------------------------------------------------------------------------------|
-| `name`                  | Model weights, default is `ViT-B/32`. Support all OpenAI released pretrained models.                                           |
+| `name`                  | Model weights, default is `ViT-B-32::openai`. Full list of models and weights can be found at [here](#model-support)                        |
```
Comment (ZiniuYu) [inline docs/user-guides/server.md:267]:
```suggestion
| `name`                  | Model weights, default is `ViT-B-32::openai`. A full list of models and weights can be found {ref}`here<model-support>`                        |
```

**Review by ZiniuYu — CHANGES_REQUESTED** [review]

**Review by ZiniuYu — APPROVED** [review]

**Review by numb3r3 — APPROVED** [review]

**Pull request merged**